recipe_batches/recipe_batches.py

```diff
@@ -3,7 +3,15 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  multiples = []
+
+  for i in recipe:
```

Comment:
Consider renaming i to ingredient to be more specific and make your code easier to understand